Documentation/PageTsconfig/Templates.rst

```diff
@@ -0,0 +1,128 @@
+.. include:: /Includes.rst.txt
+
+.. index::
+   Page TSconfig; templates
+   templates
+.. _pagetemplates:
+
+
+=========
+templates
+=========
+
+.. versionadded:: 12.0
+
+All Fluid templates rendered by backend controllers can be overridden with own
+templates on a per-file basis. The feature is available for basically all core
+backend modules, as well as the backend main frame templates. Exceptions are
+email templates and templates of the install tool.
+
+.. caution::
+
+   While this feature is powerful and allows overriding nearly any backend
+   template, **it should be used with care**: Fluid templates of the core
+   extensions are not considered API. The core development needs the freedom to
+   add, change and delete Fluid templates any time, even for bugfix releases.
+   Template overrides are similar to an XCLASS in PHP - the core can not
+   guarantee integrity on this level across versions.
+
+
+Basic syntax
+============
+
+The various combinations are best explained by example:
+
+The linkvalidator extension (its composer name is `typo3/cms-linkvalidator`)
+comes with a backend module in the :guilabel:`Web` main section. The page tree
+is displayed for this module and linkvalidator has two main views and templates:
+
+:file:`Resources/Private/Templates/Backend/Report.html` for the
+:guilabel:`Report` view and another for the :guilabel:`Check link` view. To
+override the :file:`Backend/Report.html` file with a custom template, this
+definition can be added to the :file:`Configuration/page.tsconfig` file of an
+extension:
+
+.. code-block:: typoscript
+   :caption: EXT:site_package/Configuration/page.tsconfig
+
+   # Pattern: templates."composer-name"."something-unique" = "overriding-extension-composer-name":"entry-path"
```

Comment:
I would probably move this out of the code-block or shorten it to prevent horizontal scrolling